big-practice/src/js/pages/dashboard.page.js

```diff
@@ -42,23 +43,34 @@ class Dashboard {
     DocumentHelper.cleanErrorMessage(this.nationInput);
   }
 
-  // /**
-  //  * Modal form update will appear when clicking the update button in the table
-  //  *
-  //  * @param {element} item - Table update button
-  //  */
-  // tableUpdateBtn(item) {
+  /**
+   * Modal form update will appear when clicking the update button in the table
+   *
+   * @param {element} item - Table update button
+   */
+  // async tableUpdateBtn(item) {
   //   const movieId = item.dataset.id;
 
-  //   showModal(modalForm);
-  //   getApi(`${MOVIES_API}/${movieId}`, (movieData) => {
-  //     nameMovieInput.value = movieData.name;
-  //     directorInput.value = movieData.director;
-  //     nationInput.value = movieData.nation;
-  //     form.setAttribute('data-id', movieId);
-  //   });
+  //   ModalHelper.showModal(this.modalForm);
+  //   const movieData = await apiService.get(`${MOVIES_API}/${movieId}`);
+
+  //   this.nameMovieInput.value = movieData.name;
+  //   this.directorInput.value = movieData.director;
+  //   this.nationInput.value = movieData.nation;
+  //   this.form.setAttribute('data-id', movieId);
   // }
 
+  async tableUpdateBtn(item) {
```

Comment:
What does tableUpdateBtn mean? Function should be started with a verb and describe what it really do

Reply:
Fixed